### ISSUE TITLE
Create usage for SpatialCorrelationFunction

### ISSUE THREAD

**Issue by SergejMuhic** [bSI-InfraRoom/IFC-Specification#670]
Would it not make more sense to exchange this kind of information in the project documentation? In the IFC file it serves no purpose, it is impossible to format so really hard to convey any useful information. @j-weil @Michel-Rives

_Originally posted by @SergejMuhic in https://github.com/bSI-InfraRoom/IFC-Specification/pull/658#pullrequestreview-1520711522_

**Comment by larswik** [comment]
I would like to add a usage for Dataset Association at the same time to cover for all the "LinkToData" requirements in the Pset excel sheet from the domain team.
Furthermore, a usage for Material Constituent Set should be added (see #672). However, this is registered as a separate issue since PR #669 needs to be merged first.

**Comment by pjanck** [comment]
Seems fixed by #673
